src/routes/authRoutes.ts

```diff
@@ -0,0 +1,14 @@
+import { Hono } from 'hono';
+import { validateUser } from '../middleware/userValidation';
+import { authController } from '../controllers/authController';
+import { validateSignIn } from '../middleware/authValidation';
+
+export const authRoutes = new Hono();
+
+authRoutes.post('/signup', validateUser, authController.signUp);
+
+authRoutes.post('/signin', validateSignIn, authController.signIn);
+
+authRoutes.post('/logout', authController.logOut);
+
+authRoutes.get('/refresh', authController.refresh);
```

Comment:
Same

Reply:
https://github.com/EugeneTsalko/backend-hono/pull/2#discussion_r1428043264
If we remove the refresh method, this route is meaningless and should be removed